<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TECSettingsDiag</class>
 <widget class="QDialog" name="TECSettingsDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TEC file to Raster Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>460</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>設定Raster圖層解析度(像素大小)</string>
   </property>
   <property name="text">
    <string>Set Resolution</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="resolutionInput">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="currentResLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Setting : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>設定Raster圖層的顯示套色</string>
   </property>
   <property name="text">
    <string>Set Color Scheme For TEC Attribute :</string>
   </property>
  </widget>
  <widget class="QListWidget" name="TECAttrList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>241</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>54</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>點與點的最小距離(小於此距離視為同一點)</string>
   </property>
   <property name="text">
    <string>Minimum Distance Between Points:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minDisInput">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>55</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="ColorRampLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TECSettingsDiag</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TECSettingsDiag</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
